### ISSUE TITLE
...more info buttons are ambiguous links.

### ISSUE THREAD

**Issue by dbltoe** [zencart/zencart#4940]
With just _...more info_ shown in place of the _add:_ quantity box for products with attributes, ADA/WCAG Tests show them as ambiguous links since they are all  >...more info</a> in the link.

One possibility would be to add 'products_name to the define in includes/languages/english.php BUT, this could result in a button that takes up the whole page. 

Somethng like

<a class="list-moore"href="https://yoursite.com/index.php?main_pagge=producct_info&cPagth=22&products_id=34" title="More Info for Product ID $products_id">... more info</a>

I may be putting that together wrong but, this way the buttons can stay as they are but the system should now see the different titles and not ding for ambiguity.  The title would remain short as well.

Thoughts?

**Comment by scottcwilson** [comment]
Product name is good.   Button length limit is 30 chars so no worries - use title as you suggested in your example.

**Comment by scottcwilson** [comment]
... and thank you for keeping Zen Cart friendly to people with disabilities.

**Comment by dbltoe** [comment]
Thanks.  Looks like we'll need to do the same for the Call for price.

![Screenshot 2022-06-29 114742](https://user-images.githubusercontent.com/5122886/176492298-04fcd74d-e019-43f6-8322-96d25ef723f5.png)

Together or separate?

**Comment by lat9** [comment]
My vote:  **Together**, with a change to the issue's title.

**Comment by lat9** [comment]
Try

`$link = '<a class="list-more" href="' . zen_href_link(zen_get_info_page($products_new->fields['products_id']), 'cPath=' . zen_get_generated_category_path_rev($products_new->fields['master_categories_id']) . '&products_id=' . $products_new->fields['products_id']) . ' title="' . $products_new->fields['products_id'] . '">' . MORE_INFO_TEXT . '</a>';`

That's changing the ending 'bits' after the `title="` tag, removing an extra `'"`.

**Comment by dbltoe** [comment]
I thought that area was "busy"

No partial blank but no title either.

**Comment by lat9** [comment]
Also missing the closing `"` for the anchor tag's href attribute:
`$link = '<a class="list-more" href="' . zen_href_link(zen_get_info_page($products_new->fields['products_id']), 'cPath=' . zen_get_generated_category_path_rev($products_new->fields['master_categories_id']) . '&products_id=' . $products_new->fields['products_id']) . '" title="' . $products_new->fields['products_id'] . '">' . MORE_INFO_TEXT . '</a>';`

Added a `"` just before the 'title=' attribute.

**Comment by dbltoe** [comment]
That cured the problem for the ... more info on that page except the Zeen Cart banners and I vote ignore on those.

Off to a CT scan but will address this later.  Meanwhile, if someone can point to definitive guidelines to doing changes to more than one file.....   There are nearly 20.

**Comment by scottcwilson** [comment]
Please put them all in a single PR, since the motivation is the same.

**Comment by dbltoe** [comment]
> Meanwhile, if someone can point to definitive guidelines to doing changes to more than one file.....

**Comment by dbltoe** [comment]
I believe this was fixed in #5070

**Comment by scottcwilson** [comment]
Can this issue be closed, @dbltoe ?

**Comment by scottcwilson** [comment]
Closing, please create new issue if there are outstanding issues that should be addressed.
